backend/internal/model/game.go

```diff
@@ -145,3 +148,21 @@ func (g *game) GetShotsData() ShotsData {
 
 	return g.shotsData
 }
+
+func (g *game) IncrementHeatmap(xCord float64, yCord float64) error {
```

Comment:
What is the reason for returning an `error` value if the function always returns the `nil`? 🤔

Reply:
There is a possibility that an `error` will be returned if the `xCord` or `yCord` passed to the function is out of range of the heatmap array. I can now see that the variable name `heatmapIndex` is misleading because  it's the upper bound of the heatmap. I will fix it.

Reply:
Sorry, I'm blind 😂 I didn't see these return statements before